src/Sulu/Bundle/TranslateBundle/Command/ExportCommand.php

```diff
@@ -76,50 +64,93 @@ protected function configure()
                 'sets the filename of the exported file',
                 'sulu'
             );
+
+        $this->setDescription(
+            'Loads translation messages as defined by the parameters '
+            . 'from the bundles and writes them to a file, which also can be configured '
+            . 'via the parameters. If a locale got specified, only the translation '
+            . 'corresponding to that locale get exported. Otherwise the translation '
+            . 'files get exported for all locales defined in the system.'
+        );
     }
 
+    /**
+     * Loads translation messages as defined by the parameters from the bundles
+     * and writes them to a file, which also can be configured via the parameters.
+     * If a locale got specified, only the translation corresponding to that locale
+     * get exported. Otherwise the translation files get exported for all locales
+     * defined in the system.
+     *
+     * @param InputInterface $input The input of the command
+     * @param OutputInterface $output The output of the command
+     *
+     * @return int 0 iff everything went fine
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $locale = $input->getArgument('locale');
-        $format = $input->getArgument('format');
+        $locales = [];
+        if ($locale) {
+            array_push($locales, $locale);
+        } else {
+            $locales = array_merge($locales, $this->getContainer()->getParameter('sulu_core.translations'));
+        }
+        foreach ($locales as $l) {
+            $this->executeForLocale($input, $output, $l);
+        }
+
+        return 0;
+    }
+
+    /**
+     * Executes the command for a given locale.
+     *
+     * @param InputInterface $input The input of the command
+     * @param OutputInterface $output The output of the command
+     * @param $locale string The locale
+     */
+    protected function executeForLocale(InputInterface $input, OutputInterface $output, $locale)
```

Comment:
Isn't `exportForLocale` the better name? Then it is clearer what this  method is actually doing.